optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc

```diff
@@ -169,11 +169,11 @@ but instead are added to the tail of the solver phases list.
 
 
 [[solutionFileIO]]
-=== SolutionFileIO: Input And Output Of Solution Files
+=== `SolutionFileIO`: input and output of solution files
```

Comment:
Suggestion for separate PR (so not blocking this PR):
Here ` are added, but below there are removed.

Reply:
Note that it wasn't consistent to start out with - which is something we'll want to clean up for consistency. (This PR is fine, nothing wrong with it.)